src/main/java/eod/warObject/leader/Leader.java

```diff
@@ -98,8 +100,8 @@ public void die() {
         player.loseLeader();
     }
 
-    public Card getSpecialOrder() {
-        return specialOrder.remove(0);
+    public Card getCommand() {
+        return command.remove(0);
     }
```

Comment:
Why use remove?

Reply:
Cuz there's only two command cards, and neither will be given again.

Reply:
During the initialization, two of the command cards are created, then the memory will be released as the command cards are removed from the list.

Reply:
Wrong.
According to the 9th section of the rule docs, it should not be removed because we'll use it again and again.

Quote:
> 9.指令卡
> 指令卡是獨立於手牌之外，每回合都會固定獲得的卡片，其作用方式與行動卡相同，但是會在回合開始時會回到原始狀態。
> 
> EX.在回合中使用了兩張移動指令，在下個回合開始時，會回到手中